backend/windmill-api/src/workspaces.rs

```diff
@@ -831,6 +832,21 @@ async fn edit_error_handler(
     .execute(&mut *tx)
     .await?;
 
+    if ee.error_handler.as_ref().unwrap() == slack_error_handler_path {
+        // Give read access to the 'slack_bot' folder (containing the slack bot token resource) to the created 'g/error_handler'
```

Comment:
should verify that it's not already the case otherwise will keep appending forever

Reply:
FSA I tested running this query multiple times and it doesn't append again and again. It's probably because it's treated as a dict so adding a key that already exists just updates its value. Anyways, changing the approach so this doesn't matter